src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,80 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import React from 'react';
+
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pagesAmount);
+  const isSelected = (page:number) => currentPage === page;
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', {
+        disabled: isSelected(1),
```

Comment:
You don't need to call this function each time. Just create a variable and reuse it